python/py-ansible-compat/Portfile

```diff
@@ -15,14 +15,19 @@ maintainers         {judaew @judaew} openmaintainer
 description         \
     Functions that help interacting with various versions of Ansible
 long_description    {*}${description}
-homepage            https://github.com/ansible/ansible-compat
 
-checksums           rmd160  ca04c268c763cebfd6b07551376c585b6d484138 \
-                    sha256  b3e9f9d7c3a1ce6222de444e9dc6fece7eba70ac64f2a0befdc4e2d542018b4a \
-                    size    73021
+homepage            https://pypi.python.org/pypi/ansible-compat
+master_sites        pypi:a/ansible_compat
+distname            ansible_compat-${version}
+distfiles           ansible_compat-${version}.tar.gz
+dist_subdir         ansible_compat-${version}
```

Comment:
of the four lines above only `distname` is needed; the rest is the default from the `python` PortGroup so please remove those

Reply:
Strange, when I tested originally, I was getting errors with the download due to pypi needing ansible**_**compat vs. ansible**-**compat so I set the master_sites variable.  I'm no longer getting the download error...